keyboards/maxr1998/pulse4k/config.h

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
```

Comment:
```suggestion
Copyright 2019 Max Rumpf
```
You wrote this file so should be licensed by you. It is also standard to have the licence header at the start of all of your .C and .h files

Reply:
From what I've seen most keyboards have the config licensed to him, so not sure if I should change this one. Good point on adding a header to the other files.

Reply:
So, added the headers, and modified the one in the config.h so that it makes the most sense. Would appreciate if you could take another look at it @e11i0t23. Thanks!

Reply:
Most keyboards are very copypasted, they should ideally have the original submitter's name on all of them ;)